File: Tp1/Cometa/Cometa.java
import java.util.Scanner;

public class Cometa{
	public static int cometaAno(int ano){
		int inicio = 1986;
		for(inicio = 1986; inicio <= ano; inicio += 76){
		}
		return inicio;
	}	

	public static void main(String args[]){
		Scanner entrada = new Scanner(System.in);

		int ano = entrada.nextInt();
		do{
			System.out.println(cometaAno(ano));
			ano = entrada.nextInt();
		}while(ano != 0);
	}
}

File: Tp1/Cometa/Cometa.c
#include <stdio.h>

int cometaAno(int ano){
	int inicio = 1986;
	for(inicio = 1986; inicio <= ano; inicio += 76){}

	return inicio;
}

int main(void){
	int ano = 0;
	scanf("%d", &ano);
	do{
		printf("%d\n", cometaAno(ano));
		scanf("%d", &ano);
	}while(ano != 0);	

	return 0;
}
